src/MPDevice.cpp

```diff
@@ -1773,7 +1791,7 @@ bool MPDevice::addOrphanParentToDB(MPNode *parentNodePt)
     else
     {
         /* It is important to note that this function is called with a valid linked chain (due to tagcheck) */
-        for (auto &i: loginNodes)
+        for (auto &i: list)
```

Comment:
Please use a different name for "list" here. list is often refered as std::list. also if one does something like `use namespace std;` there will be a name clash